en/developer_guide/logs.rst

```diff
@@ -0,0 +1,75 @@
+Logs
+====
+
+For logs we use Monolog. The Logs table is displayed in administration section in the backoffice. it displays the date and time, user's IP, user's name, the current site's name and the log's message.
+
+The LogBundle is in openorchestra-cmsbundle.
```

Comment:
This sentence is partially wrong.

By saying it, you put a limit on the refactoring we could do with the logBundle